RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,864 @@
+---
+openapi: 3.0.2
+info:
+  title: ASTM UAM PSU RIS API
+  description: |-
+    Interface definitions for communication between a UAS Service Supplier (USS) and a 
+    Resource Information Service (RIS). This includes APIs for resource definition, resource 
+    capacity, and resource status. 
+  version: 0.0.0
+servers: 
+- url: /
+security:
+- Authority:
+  - utm.strategic_coordination
+tags:
+- name: Resource references
+  description: Endpoints exposed by the RIS for details regarding resource references. 
+- name: Resource capacities
+  description: Endpoints exposed by the RIS for details regarding future resource capacities.
+- name: Resource definition
+  description: Endpoints exposed by the RIS for details regarding the physical definition and default capacity of a resource. 
+- name: Resource status
+  description: Endpoints exposed by the RIS for details regarding the current status and capacity of a resource. 
+- name: p2p_utm
+  description: Endpoints exposed by PSUs for peer-peer communication.
+
+paths:
+  /ris/v1/resource_details:
+    summary: RIS shares detailed information about a resource. 
+    post:
+      tags:
+      - Resource references
+      - Resource capacities
+      - Resource definition
+      - Resource status
+      - p2p_uam
+      summary: Notify a peer PSU of changed resource details.
+      description: Notify a peer PSU directly of changed resource details (usually as a requirement of previous interactions with the DSS).
+      operationId: notifyResourceDetailsChanged
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PutResourceDetails'
+        required: true
+      responses:
+        "204":
+          description: New or updated full resource information received successfully.
+        "400":
+          description: |-
+            * One or more parameters were missing or invalid.
+            * The Entity could not be parsed, or contains illegal data.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
```

Comment:
I noticed that this reference syntax ($ref) seems incorrect, and it's not the only occurrence.  Shouldn't it be "../DSS/astm-uam-psu-dss-0.0.0.yaml**#**/components/schemas/ErrorResponse' instead?